src/blocks/disk_space.rs

```diff
@@ -97,14 +108,6 @@ pub struct DiskSpaceConfig {
     /// Diskspace alert in GiB (red)
     #[serde(default = "DiskSpaceConfig::default_alert")]
     pub alert: f64,
-
-    /// Show percentage
-    #[serde(default = "DiskSpaceConfig::default_show_percentage")]
```

Comment:
Is it possible to keep this and show_bar so we don't break existing configs? We can mark it as deprecated and they can be removed in a later release. The CPU and battery blocks do something like that.

Reply:
Sure, I guess we could override the default format string in this case, unless it is manually set.

Reply:
Yeah that work work